<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNewAnagrafica</class>
 <widget class="QWidget" name="HNewAnagrafica">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuova anagrafica</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/Resources/User-Male.PNG</normaloff>:/Resources/User-Male.PNG</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>*Ragione sociale:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leRagsoc">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Indirizzo:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leAddress">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Citt√†:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leCity">
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>CAP:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leCap">
         <property name="maxLength">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Provincia/stato:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leProvincia"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Telefono:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="leTel">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Contatto:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="leContatto">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Note:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="leNote"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Nazione:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leNazione">
         <property name="maxLength">
          <number>45</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="cbCliente">
         <property name="text">
          <string>Cliente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbfornitore">
         <property name="text">
          <string>Fornitore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbTrasportatore">
         <property name="text">
          <string>Trasportatore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSubcliente">
         <property name="text">
          <string>Subcliente</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Salva</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/save-icon.png</normaloff>:/save-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Chiudi</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/Actions-window-close-icon.png</normaloff>:/Actions-window-close-icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leRagsoc</tabstop>
  <tabstop>leAddress</tabstop>
  <tabstop>leCity</tabstop>
  <tabstop>leCap</tabstop>
  <tabstop>leProvincia</tabstop>
  <tabstop>leNazione</tabstop>
  <tabstop>leTel</tabstop>
  <tabstop>leContatto</tabstop>
  <tabstop>leNote</tabstop>
  <tabstop>cbCliente</tabstop>
  <tabstop>cbfornitore</tabstop>
  <tabstop>cbTrasportatore</tabstop>
  <tabstop>cbSubcliente</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
